torchbox/torchbox.com
# Remove unneeded fields on historical work page

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/views/6453316)

### Description of Changes Made

A follow-on tidy up from investigating the issues with crashing pages on the production server. The base cause was very large images - but this removes some unnecessary image fields on historical work pages. I checked a few historical case studies and most seemed to use both the homepage image field and the feed image field - so I've updated the `listing_image` property to use the feed image, as that is more intuitive.

### How to Test

Check that the work listing page still displays images
Edit a historical work page and you will no longer see the 'screenshots' or 'homepage image' fields.

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
